joss_paper/paper.md

```diff
@@ -424,6 +424,9 @@ aas-doi: Add DOI HERE
 aas-journal: The Astrophysical Journal
 ...
 
+[^*]: The author list in this paper is sorted by number of commits to the core
```

Comment:
I would not use the word "core" which is unclear. Why not just say "sunpy package"?

Reply:
"core sunpy repository" makes it pretty clear doesn't it? "package" isn't really the right term here because we are talking about the git repo.

Reply:
Should we say for master only as well?